runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscoreConfig.java

```diff
@@ -45,4 +45,37 @@ default boolean playerOption()
 	{
 		return true;
 	}
+	
+	@ConfigItem(
```

Comment:
Isn't this unnecessary? You would want to have these options everywhere or nowhere.

Reply:
It might be unnecessary, but some people might want to remove some of them. I will remove these config options as I cannot nicely differentiate between the options due to the way I add them.